platformio.ini

```diff
@@ -96,6 +96,7 @@ build_flags =
 	-DOXRS_RACK32
 	; TFT_eSPI configuration
 	-DUSER_SETUP_LOADED=1
+  -DDISABLE_ALL_LIBRARY_WARNINGS=1
```

Comment:
Can you fix the spacing? Or is it just my mobile rendering this wrong?

Reply:
was spacee instead of tab